ArduCopter/ArduCopter.cpp

```diff
@@ -485,7 +485,10 @@ void Copter::one_hz_loop()
 
         update_using_interlock();
 
-#if FRAME_CONFIG != HELI_FRAME
+#if FRAME_CONFIG == HELI_FRAME
+        // check if we are using governor enable on an aux switch
+        set_using_governor_switch(check_if_auxsw_mode_used(AUXSW_HELI_GOV_ENABLE));
```

Comment:
The intent here is that a user will not need to have an enable switch assigned once they are sure the system is working correctly.  So if they are not using a switch, we need to know.  If that is the case, then the governor will come off/on simply with the RPM sensor being healthy.